bindings/python/native/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
```

Comment:
I would add [workspace] above here so we don't need it in the root cargo.toml of the repo

Reply:
Agree. Modified.